course6/toyapp_mikhail/displaylib/triangle.h

```diff
@@ -0,0 +1,21 @@
+#pragma once
+
+#include "vector3d.h"
+
+#include <array>
+
+namespace Display {
+
+	class Triangle
+	{
+	private:
+		std::array<Vector3D,3> _vertex;
```

Comment:
You didn't use the _ prefix in other places, and you left the "m" prefix in another. Ugly...

The prefix is OK, but the C and C++ standard has a note that all identifiers starting with _ are reserved for internal use, so some people prefer to use _ suffix instead.